File: face/geomajas-face-dojo/client/src/main/resources/geomajas/map/print/ImageComponentInfo.js
dojo.provide("geomajas.map.print.ImageComponentInfo");
/*
 * This is part of Geomajas, a GIS framework, http://www.geomajas.org/.
 *
 * Copyright 2008-2011 Geosparc nv, http://www.geosparc.com/, Belgium.
 *
 * The program is available in open source according to the GNU Affero
 * General Public License. All contributions in this program are covered
 * by the Geomajas Contributors License Agreement. For full licensing
 * details, see LICENSE.txt in the project root.
 */
dojo.declare("ImageComponentInfo", PrintComponentInfo, {

	/**
	 * @class 
	 * A map component (mirror of server object)
	 * @author Jan De Moerloose
	 *
	 * @constructor
	 */
	constructor : function () {
		this.javaClass = "org.geomajas.plugin.printing.component.dto.ImageComponentInfo";
	},

	getImagePath : function () {
		return this.imagePath;
	},
	
	setImagePath : function (imagePath) {
		this.imagePath = imagePath;
	}

});

File: plugin/geomajas-plugin-printing/printing/src/main/java/org/geomajas/plugin/printing/component/dto/ImageComponentInfo.java
/*
 * This is part of Geomajas, a GIS framework, http://www.geomajas.org/.
 *
 * Copyright 2008-2012 Geosparc nv, http://www.geosparc.com/, Belgium.
 *
 * The program is available in open source according to the GNU Affero
 * General Public License. All contributions in this program are covered
 * by the Geomajas Contributors License Agreement. For full licensing
 * details, see LICENSE.txt in the project root.
 */
package org.geomajas.plugin.printing.component.dto;

import org.geomajas.annotation.Api;

/**
 * DTO object for ImageComponent.
 * 
 * @author Jan De Moerloose
 * @see org.geomajas.plugin.printing.component.impl.ImageComponentImpl
 * @since 2.0.0
 */
@Api(allMethods = true)
public class ImageComponentInfo extends PrintComponentInfo {

	private static final long serialVersionUID = 200L;

	private String imagePath;

	/**
	 * Get the image path.
	 *
	 * @return image path
	 */
	public String getImagePath() {
		return imagePath;
	}

	/**
	 * Set the image path.
	 *
	 * @param imagePath image path
	 */
	public void setImagePath(String imagePath) {
		this.imagePath = imagePath;
	}

}
